app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java

```diff
@@ -142,17 +142,7 @@ private String getFormattedWikiText(final String wikiText, final String editedLo
 
             return wikiText + editedLocation;
         }
-        if (!wikiText.contains("filedesc") && !wikiText.contains("Location")) {
```

Comment:
Not sure if this was intended or a mistake, let me know 😅

Reply:
@Ayan-10 I believe you wrote this class, could you comment on this please? :)

Reply:
I was doing case by case comparison at that time that's why I wrote like that. But now it seems redundant. Sorry for not recognizing it earlier.

Thanks